src/Components/Paginator.js

```diff
@@ -78,7 +78,7 @@ class Paginator extends Component {
         }
 
         return (
-            <Pagination size="sm" style={{float: "right", marginBottom: "0rem"}}>
+            <Pagination size="sm" style={{ marginBottom: "0rem"}}>
```

Comment:
What's the goal of removing this attribute? On my browser, it now puts the paginator and label on top of each other instead of one in the left corner and one in the right.
![float_right_1](https://github.com/user-attachments/assets/67355c1e-4490-46d6-85e3-6cc91be02715)
![float_right_2](https://github.com/user-attachments/assets/9cba8e0d-10bd-4d45-abe2-e6aa4ccc461d)

Reply:
It is better for responsive, the overall idea of this pr is to have the minimum to be able to see the data on mobile. The idea is to create a second iteration where we should redesign some of the basic components and make it look nicer and create a better responsive for tables.